trino/client.py

```diff
@@ -62,6 +62,7 @@
 
 import trino.logging
 from trino import constants, exceptions
+from trino._version import __version__
```

Comment:
having version in `trino/__init__.py` would cause a circular dependency for the following import statement:

```python
from trino import __version__
```

So I ended up moving to another submodule called `_version`. Much more convenient to be accessed by `setup.py` and insider `trino`.

Reply:
why exactly name `_version` ? could you please point to some docs that describes that rule?

Reply:
it could be `__version__.py` as well later to be overwrite in `__init__.py` with `from .__version__ import __version__` like how `requests` does it.

I choose `_version.py` as a "private" package path. This is kinda pseudo convention and it's not a strict rule. I [searched](https://github.com/search?q=path%3A**%2F_version.py+language%3APython&type=code&ref=advsearch) on Github for `_version.py` and found 18k matches. 

I can change it to `__version__.py` if you prefer so.

Reply:
Thanks, I don't have a strong preference here, just was looking for some "official" rule. 

@hashhar WDYT? I believe these changes should be extracted to a separate commit.

Reply:
Yes, this setup.py makes sense to extract.

Sorry I didn't notice initially.

Reply:
Either filename is fine by me. The _version_.py does however give the feeling it's something special.